hazelcast-spring/src/test/resources/com/hazelcast/spring/node-client-applicationContext-hazelcast.xml

```diff
@@ -22,7 +22,8 @@
        xsi:schemaLocation="http://www.springframework.org/schema/beans
 		http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
 		http://www.hazelcast.com/schema/spring
-		http://www.hazelcast.com/schema/spring/hazelcast-spring.xsd">
+		hazelcast-spring-3.9.xsd
```

Comment:
It's the same we use in `fullConfig-applicationContext-hazelcast`. And this one actually works for me. Although `hazelcast-spring.xsd` is set to 3.9 in `spring.schemas`, it doesn't seem to be resolved correctly. So the XML is full or errors. After using the same schema location as in the other XML, it works for me. If this can be fixed in a better way, I'd be happy to see how.